### DISCUSSION TITLE
2024-04-23 Meeting Agenda

### DISCUSSION THREAD

**Discussion by aflackey** [ucum-org/ucum#305] [Meetings]
### Date: 2024-04-23, 4:00 – 5:00 pm EST
**Chair(s): Marjorie Rallins**

|**TIME** |**AGENDA TOPICS** |**LEAD(S)** |
| ----------- | -----| -----|
|4:00 - 4:05 | Welcome & opening remarks | Marjorie Rallins |
|4:05 - 4:10 | Update on user stories for Licensing Discussion|Tim Briscoe|
|4:10-4:30|Input into attributes/properties for new units|Eza Hafeza, All|
|4:30-4:50|Annotations| Eza Hafeza, All |
|4:50 - 4:00 | New and Open Proposals (as time permits) | All
|5:00 | Adjourn | Eza Hafeza |

***OPEN ACTION ITEMS***
|**ITEM** |**DATE IDENTIFIED**|**TARGET COMPLETION**| **LEAD** |
| ----------- | -----| -----| -----|
| Continue asynchronous discussion on conformance / validation, [UCUM validation #250](https://github.com/orgs/ucum-org/discussions/250) | 2023-11-28 | Post first release | All |
| Collect use cases / user stories that may help inform license discussions | 2023-07-25 | 2023-11-14 | Tim Briscoe et. al. |

***LINKS TO REFERENCE MATERIALS***
[UCUM Committee_Meeting Summary_2024-03-27_FINAL](https://docs.google.com/document/d/1Tuh8G9x9NGzvJ0UcSHNWqSbcglYOL_Sa/edit)

[UCUM New Unit Template_2024-01](https://docs.google.com/spreadsheets/d/1xBGaNLuMaJBFnFdi81ezDF5oHzN_Ozww/edit?usp=drive_link&ouid=110875911742539584915&rtpof=true&sd=true)

[All active GutHub issues/proposals](https://github.com/ucum-org/ucum/issues)

@ucum-org/regenstrief 
@ucum-org/advisors

**Comment by ucumworker** [comment]
-- Please reply above this line --

Hello Ucum-Org/ucum,

We received your email request and will get back to you as soon as we
can. Please know our response time does vary at different times of the
year due to our bi-annual release. 

Thank you,

Best Regards,
The LOINC Team
